Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift

```diff
@@ -27,6 +27,11 @@ extension ConnectionConfiguration {
             clientOptions.environment = environment
         }
         
+        if let remainPresentForMilliseconds = remainPresentForMilliseconds {
```

Comment:
Could we add a test to `Tests/InternalTests/Configuration/ConnectionConfiguration+ExtensionsTests.swift` which tests that it correctly sets the `transportParams` when `remainPresentInMilliseconds` is specified?

Reply:
Good point, added in 06526ce6fd20524221d48851b6b26da65f80026c

Reply:
Good point, added in 06526ce6fd20524221d48851b6b26da65f80026c